### PR TITLE
Added the code for Linear Search (Python)

### LINKED ISSUE #602: Linear Search
Hi I am Malavika R Vikraman , participating in GSSOC'21 .
I would like to contribute by adding Linear Search in Python to this project.

Kindly assign this issue to me!
@anubhavitis @dheerajkotwani @tarun26091999 @todi-2000 @geekquad @plazzy99

### PR DESCRIPTION
**Pull request by malavikarajeshvikraman**
Hi, I have added the code for Linear Search in python ( Linear Search in Python  (fix #602 ) ) .
 Please review and merge it  @tarun26091999 .

### CODE REVIEW

File: Algorithm/Searching_Sorting/readme.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ is one of the techniques for making the elements ordered.
 
 Following are the types of searches which we will be discussing in this book.
 
-* Unordered Linear Search
+* Unordered Linear Search ----> [Python](/Code/Python/linearsearch.py)
```
Comment (rudrakshi99) [inline Algorithm/Searching_Sorting/readme.md:53]: add this in the question section.

File: Code/Python/linearsearch.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+itemFound = linearSearch(item,l)
+if itemFound:
+    print ('True')
+else:
+    print ('False')
+ 
+'''
+ 
+Test Case :
```
Comment (rudrakshi99) [inline Code/Python/linearsearch.py:27]: add at least 2 test cases.

File: Code/Python/linearsearch.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+ 
+Input : 100 56 789 456 321
+        789
+ 
+Output  : True 
+ 
+Time Complexity: O(n)
+
```
Comment (rudrakshi99) [inline Code/Python/linearsearch.py:35]: add space complexity also.

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

**Comment by malavikarajeshvikraman** [comment]
@rudrakshi99  I have made the changes you requested. Please do merge it if its alright !

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
space complexity is wrong. Add comments to explain the steps.

**Review by rudrakshi99 — APPROVED** [review]

**Review by tarun26091999 — APPROVED** [review]

**Pull request merged**